src/ai/backend/manager/models/agent.py

```diff
@@ -317,6 +323,10 @@ async def batch_load(
             )
 
 
+class ModifyAgentInput(graphene.InputObjectType):
+    schedulable = graphene.Boolean(required=True, default=True)
```

Comment:
Let's make this field optional. Other fields of the `agents` table should be updated without altering the value of `schedulable`. For example, I want to add another mutable field, `scaling_group` here, in a separate issue in the near future.